src/cmd/src/datanode.rs

```diff
@@ -96,6 +96,8 @@ struct StartCommand {
     #[clap(long)]
     data_home: Option<String>,
     #[clap(long)]
+    wal_dir: Option<String>,
+    #[clap(long)]
```

Comment:
This requires an update to the docs

Reply:
en: https://github.com/GreptimeTeam/docs/blob/main/docs/en/v0.4/user-guide/operations/configuration.md
zh: https://github.com/GreptimeTeam/docs/blob/main/docs/zh/v0.4/user-guide/operations/configuration.md